wordpress-mobile/WordPress-iOS
# Unified About: Add Rate Us option

Implements #17420. This PR adds the Rate Us option to the Unified About screen.

<img src="https://user-images.githubusercontent.com/4780/143265624-54457229-123e-49c1-82ea-3172db8e4a35.jpeg" width="350">

**To test**

* Build and run
* Navigate to About, then tap Rate Us
* Ensure you're taken to the App Store listing for WordPress and see the add a review UI shown above

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
